document_page_quality_manual/README.rst

```diff
@@ -54,6 +54,7 @@ Contributors
 * Odoo SA <info@odoo.com>
```

Comment:
Can't select the exact lines but there are 9.0 to change to 10.0 on lines 31 and 45